tests/python/gaia-ui-tests/gaiatest/tests/functional/lockscreen/manifest.ini

```diff
@@ -26,3 +26,5 @@ carrier = true
 [test_lockscreen_notification.py]
 
 [test_lockscreen_wake_with_notification.py]
+
+[test_lockscreen_time_check.py]
```

Comment:
You also need to add  ```carrier = true ```, because the automatic time setting depends on a carrier connection.